### PR TITLE
Updated frontend development setup section in readme

### LINKED ISSUE #386: Improve readme
## Description
Add instructions to set up frontend with `npm` to improve the development experience.

### PR DESCRIPTION
**Pull request by khabdrick**
## Summary
Added updates to the frontend development setup, so the developer can choose between using `npm` or the Docker container
closes #386 
## Changes
README.md

### CODE REVIEW

**Comment by NdibeRaymond** [comment]
I want to merge this now, but you should first create an issue and request that it be assigned to you. That'd help your contribution status

**Comment by NdibeRaymond** [comment]
also squash your commits with git rebase. It's a minor change so no need to have multiple commits here

**Comment by khabdrick** [comment]
@NdibeRaymond sorry I made this change directly on GitHub and I am not able to rebase.

File: README.md [hunk 1]
```diff
@@ -29,8 +29,19 @@ Follow the instructions below to set up your local development environment
...
 
 ### Frontend
-- Create a file named .env in the frontend root folder (same directory with package.json)
-- Copy the content of .env.example and paste into the new .env file.
+- Create a file named `.env` in `./zubhub/zubhub_frontend/zubhub` directory (same directory with package.json).
+- Copy the content of `.env.example` and paste into the new `.env` file.
+- On your terminal/command line, navigate to `./zubhub/zubhub_frontend/zubhub` directory.
+
+#### Using `npm`
+This is advisable for development experience.
```
Comment (tuxology) [inline README.md]: Maybe add this? 

> :information_source: This is advisable for a better development experience

**Review by tuxology — CHANGES_REQUESTED** [review]
Minor changes. Rest LGTM :rainbow_flag:

**Review by tuxology — APPROVED** [review]
Thanks, we'll merge this soon! :+1:

**Pull request merged**